de.dlr.sc.virsat.model.extension.statemachines/concept/concept.concept

```diff
@@ -7,15 +7,17 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  *******************************************************************************/
-Concept de.dlr.sc.virsat.model.extension.statemachines displayname "State Machines" description "Concept for modeling State Machines"  version 1.2 beta {
+Concept de.dlr.sc.virsat.model.extension.statemachines displayname "State Machines" description "Concept for modeling State Machines"  version 1.3 beta {
 	Import de.dlr.sc.virsat.model.extension.ps.*; 
 	
 	Category State description "State of the component at a given time"{
+		StringProperty detail description "Optional additional detail information about the state";
 	}	
 	
 	Category Transition description "Transitions among states" { 
 		Reference stateFrom of Type State description "The Transition starts in this state";
 		Reference stateTo of Type State description "The Transition ends in this state";
+		Reference trigger of Type GenericCategory description "The event that triggers the Transition";

```

Comment:
SHould a trigger be of type generic? I think we are sacrifying some semantics for the sake of simplicity. I think we should at least provide a simple Category such as a generic Event or Trigger that can be referenced and used in other Concepts.